dist/infragram.js

```diff
@@ -1679,6 +1699,49 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
     };
   }, {}],
   17: [function (require, module, exports) {
+    module.exports = function Saving(options) {
```

Comment:
So here, I believe this whole section was actually removed in https://github.com/publiclab/infragram/pull/268, so I think we can leave this out.